content/blog/2016-07-23-pull-does-not-scale-or-does-it.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Pull doesn't scale - or does it?
+created_at: 2016-07-23
+kind: article
+author_name: Julius Volz
+---
+
+Let's talk about one particularly hard-to-kill myth again. Whenever there is a
+discussion about monitoring systems and Prometheus's pull-based metrics
+collection approach comes up, someone inevitably chimes in about how a
+pull-based approach just “fundamentally doesn't scale”. When asked why they
+think this is the case, they either respond with vague fears or cite reasons
```

Comment:
Yeah. Sorry. These new github links are confusing sometimes.

On Tue, Jul 26, 2016 at 12:31 PM, Julius Volz <notifications@github.com>
wrote:

> In content/blog/2016-07-23-pull-does-not-scale-or-does-it.md
> <https://github.com/prometheus/docs/pull/499#discussion_r72286410>:
>
> > @@ -0,0 +1,178 @@
> > +---
> > +title: Pull doesn't scale - or does it?
> > +created_at: 2016-07-23
> > +kind: article
> > +author_name: Julius Volz
> > +---
> > +
> > +Let's talk about one particularly hard-to-kill myth again. Whenever there is a
> > +discussion about monitoring systems and Prometheus's pull-based metrics
> > +collection approach comes up, someone inevitably chimes in about how a
> > +pull-based approach just “fundamentally doesn't scale”. When asked why they
> > +think this is the case, they either respond with vague fears or cite reasons
>
> I have a vague fear that you were looking at the old revision of the text,
> where the initial comment was made on. Check the new one.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/prometheus/docs/pull/499/files/5542904443aaacc475e92ae9d0e4997946532ad0#r72286410>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAANaCPAQftrEww4sWn3EAOX6E3Wfvt1ks5qZjZMgaJpZM4JTa12>
> .
>